### PR TITLE
chore: add updated description and image to the enhanced-link-dialog-plugin

### PR DESCRIPTION
**Pull request by gearsdigital**
After updating my Plugin [Enhanced Toolbar Link Dialog](https://github.com/gearsdigital/enhanced-toolbar-link-dialog) I changed the Plugin UI and description slighly. This MR will update the plugin site with the values.

### CODE REVIEW

File: content/plugins/gearsdigital/enhanced-toolbar-link-dialog/plugin.txt [hunk 1]
```diff
@@ -14,4 +14,4 @@ Subcategory: enhancement
 
 ----
 
-Description: The Enhanced Toolbar Link Dialog makes it easy to add internal or external references to your content by extending the default link dialog of Kirby's textarea fields
+Description: Extend Kirby's default link dialog with a search functionality to create links to existing or external pages right at your fingertips.
```
Comment (distantnative) [inline content/plugins/gearsdigital/enhanced-toolbar-link-dialog/plugin.txt:17]: I like the rephrasing except that now it doesn't mention anymore that this is in context of the textarea field. Would be good to make that relation.

Comment (texnixe) [reply content/plugins/gearsdigital/enhanced-toolbar-link-dialog/plugin.txt:17]: @distantnative The point of the new version is that it is no longer limited to the textarea field.

Comment (distantnative) [reply content/plugins/gearsdigital/enhanced-toolbar-link-dialog/plugin.txt:17]: Ok, got it :D

Comment (gearsdigital) [reply content/plugins/gearsdigital/enhanced-toolbar-link-dialog/plugin.txt:17]: @texnixe, correct, that was my intention. Thanks for accepting this MR!

**Pull request merged**